### PR TITLE
:sparkles: Add team invite links

### PR DESCRIPTION
**Pull request by acco**
We don't support sending email in self-hosted Sequin yet. By adding team invite link support, we make it easy for users to invite other team members to their Sequin team.

### CODE REVIEW

**Comment by acco** [comment]
* **#690** <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/690?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/sequinstream/sequin/690?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

**Comment by acco** [comment]
CleanShot 2024-12-04 at 16 14 00@2x
CleanShot 2024-12-04 at 16 13 29@2x

File: lib/sequin/accounts/user_token.ex [hunk 1]
```diff
@@ -17,8 +17,12 @@ defmodule Sequin.Accounts.UserToken do
   @session_validity_in_days 60
   @impersonate_validity_in_days 1
   @account_invite_validity_in_days 7
+  @team_invite_validity_in_days 7
+  @team_invite_active_in_days 1
```
Comment (RTLS) [inline lib/sequin/accounts/user_token.ex:21]: current

File: priv/repo/migrations/20241204235903_add_hashed_token_to_user_tokens.exs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+defmodule Sequin.Repo.Migrations.AddHashedTokenToUserTokens do
+  use Ecto.Migration
+  @config_schema Application.compile_env(:sequin, [Sequin.Repo, :config_schema_prefix])
+
+  def change do
```
Comment (RTLS) [inline priv/repo/migrations/20241204235903_add_hashed_token_to_user_tokens.exs:5]: maybe add constraints?

**Review by RTLS — APPROVED** [review]

**Pull request merged**